composer.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "terminal42/contao-tablelookupwizard",
   "description": "Contao widget to select records from a foreign table. Useful if you have a large set of records.",
+  "version": "3.2.1",
```

Comment:
Never use the `version` key please :)

Reply:
What's the reason behind that ? I guess it's because Packagist does also use VCS so it doesn't need it and create "conflict". (VCS tags should does the job)

Reply:
Exactly, VCS does the job.